app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -734,6 +745,7 @@ public void loadUrl(final String url) {
     public void reload() {
         final IWebView webView = getWebView();
         if (webView != null) {
+            webView.setBlockingEnabled(Settings.getInstance(this.getContext()).shouldUseSpeedMode());
```

Comment:
Is it because sometimes the onSharedPreferenceChanged is not executed before we refresh?